flow360/component/simulation/models/volume_models.py

```diff
@@ -338,7 +337,7 @@ class BETDisk(Flow360BaseModel):
         + "torque coefficients :math:`C_t` and :math:`C_q`, defined in :ref:`betDiskLoadingNote`.",
     )
     blade_line_chord: LengthType.NonNegative = pd.Field(
-        0 * u.m,
```

Comment:
Why is the unit removed? I think your previous modification to add the unit is correct.

Reply:
fixed